pages/index.jsx

```diff
@@ -1,10 +1,20 @@
 var React = require('react');
 
+var Habitat = require('habitat');
+Habitat.load();
+var env = new Habitat();
+
+var cookieDomain = env.get('FULL_SUBDOMAIN_FOR_COOKIE');
+
 var Index = React.createClass({
   render: function() {
     return (
       <html>
         <head>
+          <script>
+            window['optimizely'] = window['optimizely'] || [];
+            window['optimizely'].push(["setCookieDomain", {{cookieDomain}}]);
+          </script>
```

Comment:
Ahhh, I see. So optimizely sets the cookie domain to mozilla.org by default. laaaaaaame